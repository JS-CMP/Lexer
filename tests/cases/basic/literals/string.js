var a = "hello";
var b = 'world';
var c = 'escape "quotes"';
