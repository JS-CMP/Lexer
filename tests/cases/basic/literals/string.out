#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {JS::Any a, b, c;a=JS::Any(u"hello");b=JS::Any(u"world");c=JS::Any(u"escape \\"quotes\\"");}
